presto-docs/src/main/sphinx/release/release-335.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Release 335 (XX May 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix failure when :func:`reduce_agg` is used as a window function. (:issue:`3883`)
+* Fix incorrect cast from ``TIMESTAMP WITHOUT TIME ZONE`` to ``TIME`` type. (:issue:`3848`)
+* Improve optimized local scheduling with regard to non-uniform data distribution. (:issue:`3922`)
+* Add support for variable-precision ``TIMESTAMP WITHOUT TIME ZONE`` type. (:issue:`3783`)
+* Return ``VARCHAR`` type when :func:`substr` argument is ``CHAR`` type. (:issue:`3599`)
+* Add a variant of :func:``substring`` that takes a ``CHAR`` argument. (:issue:`3949`)
+
+Security Changes
+----------------
+
+* Add  ``information_schema.role_authorization_descriptors`` table that returns information about what roles where granted to what principals. (:issue:`3535`)
+* Adding schema access rules to file based system access control. (:issue:`3766`)
+
+Hive Connector Changes
+----------------------
+
+* Fix a bug that caused missing column statistics after ``INSERT`` to new table. (:issue:`2469`)
+* Add support for ``information_schema.role_authorization_descriptors`` table when ``hive.security=sql-standard``.  (:issue:`3535`)
+* Allow non-lowercase column names in ``system.sync_partition_metadata`` procedure. This can be enabled
```

Comment:
docs?

Reply:
@findepi, do we have docs for this that we can link to?

Reply:
yes, this is documented: https://github.com/prestosql/presto/pull/3895
(will show up at https://prestosql.io/docs/current/connector/hive.html#procedures)

Reply:
Then I suggest to change this to link there and remove most of the detail in the release notes here.

Reply:
Please change this to a link

Reply:
And remove the second sentence, or just leave it there?

Reply:
It’s fine either way.